release/README.md

```diff
@@ -33,6 +33,8 @@ https://semver.org/
 
     When a released is published the public [user-facing release notes](https://github.com/elastisys/compliantkubernetes/blob/main/docs/release-notes/ck8s.md) needs to be updated. The new release needs to be added and the list can be trimmed down to only include the supported versions.
 
+    Create two new branches `release-vXY.Z` and `<personal-tag>/release-vXY.Z`. Add the release notes to `<personal-tag>/release-vXY.Z` and then create two PRs. One to `release-vXY.Z` and one to `main`.
```

Comment:
```suggestion
    Create two new branches `release-vX.Y.Z` and `<personal-tag>/release-vX.Y.Z`. Add the release notes to `<personal-tag>/release-vX.Y.Z` and then create two PRs. One to `release-vX.Y.Z` and one to `main`.
```
Or do we just write `vX.Y` as you have done for v0.26?

And from where do we base the branches?